app/frontend/components/notes_tagged_verse_export.js

```diff
@@ -0,0 +1,128 @@
+   If not, see <http://www.gnu.org/licenses/>. */
+
+const exportController = require('../controllers/export_controller.js');
+
+/**
+ * The TaggedVerseExport component implements the export of tagged verses into a Word document.
+ * 
+ * @category Component
+ */
+class NotesTaggedVerseExport {
+
+  enableExportButton(tabIndex, type='TAGS') {
+    var currentVerseListMenu = app_controller.getCurrentVerseListMenu(tabIndex);
+    var exportButton = currentVerseListMenu.find('.export-tagged-verses-button');
+    exportButton.removeClass('ui-state-disabled');
+    exportButton.unbind('click');
+    exportButton.bind('click', (event) => {
+      if (!$(event.target).hasClass('ui-state-disabled')) {
+        this._runExport(type);
+      }
+    });
+    exportButton.show();
+    exportButton.removeClass('events-configured');
+    uiHelper.configureButtonStyles('.verse-list-menu');
+  }
+
+  disableExportButton(tabIndex=undefined) {
+    if (tabIndex !== undefined) {
+      const currentVerseListMenu = app_controller.getCurrentVerseListMenu(tabIndex);
+      currentVerseListMenu.find('.export-tagged-verses-button').addClass('ui-state-disabled');
+    } else {
+      $('#export-tagged-verses-button').addClass('ui-state-disabled');
+    }
+  }
+
+  /**
+   * exports to Word
+   * @param {'NOTES'|'TAGS'} type 
+   */
+  async _runExport(type) {
+    /**@type {import('../ui_models/tab')} */
+    var currentTab = app_controller.tab_controller.getTab();
+    var fileName;
+    var isLoadingWholeBook = false;
+
+    if (type === 'TAGS') {
+      fileName = getUnixTagTitleList(currentTab);
+    } else if (type === 'NOTES') {
+      isLoadingWholeBook = await app_controller.translation_controller.isInstantLoadingBook(currentTab.getBibleTranslationId(), currentTab.getBook());
+      fileName = currentTab.getBookTitle() + (isLoadingWholeBook ? '' : `_${currentTab.getChapter()}`);
+    } else {
+      console.log('Unrecognized export type:', type);
+      return;
+    }
+
+    exportController.showSaveDialog(fileName).then(filePath => {
+      if (filePath) {
+        if (type === 'TAGS') {
+          renderCurrentTagsForExport(currentTab);
+        } else if (type === 'NOTES') {
+          renderCurrentChapterNotesForExport(currentTab, isLoadingWholeBook);
+        }
+      }
+    });
+  }
+  
+}
+
+module.exports = NotesTaggedVerseExport;
+
+function renderCurrentTagsForExport(currentTab) {
+  const currentTagIdList = currentTab.getTagIdList();
+  
+  const currentTagTitleList = currentTab.getTagTitleList();
+  const title = `${i18n.t("tags.verses-tagged-with")} _${currentTagTitleList}_`;
+
+  app_controller.text_controller.requestVersesForSelectedTags(
+    undefined,
+    null,
+    currentTagIdList,
+    (verses, bibleBooks, groupedVerseTags) => { 
+      exportController.saveWordDocument(title, verses, bibleBooks); 
+    },
+    'docx',
+    false
+  );
+}
+
+function renderCurrentChapterNotesForExport(currentTab, isWholeBook=false) {
```

Comment:
So, since this function can also export the whole book, we should rename it to `renderNotesForExport`